setup.py

```diff
@@ -28,8 +37,9 @@ def run_tests(self):
       version='0.1',
       description='',
       url='http://github.com/ctk3b/misibi',
-      author='Christoph Klein, Timothy C. Moore', 'Davy Yue'
-      author_email='christoph.klein@vanderbilt.edu, timothy.c.moore@vanderbilt.edu', 'davy.yue@vanderbilt.edu'
+      author='Christoph Klein, Timothy C. Moore'
+      author_email='christoph.klein@vanderbilt.edu, timothy.c.moore@vanderbilt.edu'
+      contributors='Davy Yue'
```

Comment:
Have you tested this? I don't think this is a valid keyword and is not what I was suggesting. Sorry if I was unclear. By contributors, I just meant that you are a contributor to the package which is picked up here https://github.com/mosdef-hub/msibi/graphs/contributors

http://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=metadata#metadata